CMakeLists.txt

```diff
@@ -1,5 +1,23 @@
-cmake_minimum_required(VERSION 3.0)
-project(Sopt CXX)
+cmake_minimum_required(VERSION 3.13.4) # CMake 3.13.4 is the currently default in Debian 10 repositories
+project(Sopt
+        DESCRIPTION "Sparse OPTimisation using state-of-the-art convex optimisation algorithms."
+        HOMEPAGE_URL "https://github.com/sopt/sopt"
```

Comment:
This gives me 404, should it be https://github.com/astro-informatics/sopt/ or http://astro-informatics.github.io/sopt/?

Reply:
Ah. Indeed you are right. I shall fix this and update soon.

Reply:
Fixed. Thank you for pointing this out.